ftw/simplelayout/aliasblock/browser/aliasblock.py

```diff
@@ -1,7 +1,33 @@
 from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
 from ftw.simplelayout.browser.blocks.base import BaseBlock
+from ftw.simplelayout.interfaces import ISimplelayoutBlock
+from zExceptions import Unauthorized
+
+
+try:
```

Comment:
Not needed if you look at https://github.com/4teamwork/ftw.simplelayout/blob/430e8e20e8bb84122b4e3e284e17553c51fce5ce/ftw/simplelayout/tests/test_multiview_block_properties.py